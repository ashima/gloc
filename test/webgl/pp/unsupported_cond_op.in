#if 0,1
#define X
#else
#define Y
#endif
