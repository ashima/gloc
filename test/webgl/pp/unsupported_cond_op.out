File 0, line 1, col 5 - 6:
"," is not supported in preprocessor expressions

Fatal: unrecoverable preprocessor error (5)
